docker/kabanero-utils/scripts/insecure_registry_setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # This scrip is fetching the values of 'registries.insecure' from the 'image.config.openshift.io/cluster' resource
-# and generating a script 'insecure_registry_setup.sh' that will be used by the tasks for setting the
-# 'registries.insecure' in '/etc/containers/registries.conf' file of the contianer. 
+# that will be used by the tasks for setting the
+# 'registries.insecure' in '/etc/containers/registries.conf' file of each step container in the pipelines. 
```

Comment:
In trying to keep the scripts, that are all related to pipelines, grouped in github with the other pipeline logic (yaml),  I think you should just move the entire docker folder under the pipelines folder.     

I don't think this grouping of having the scripts away from the yaml makes sense.

Reply:
Applied the change in PR : https://github.com/kabanero-io/kabanero-pipelines/pull/349